src/components/form/formcontrol/formcontrol.scss

```diff
@@ -8,6 +8,11 @@
     width: 100%;
   }
 
+  .mykn-stackctx:has(.mykn-icon) .mykn-input,
+  .mykn-stackctx:has(.mykn-icon) .mykn-select {
+    padding-right: 30px;
```

Comment:
This looks like a better approach:

can we however:

- Move this to `stackctx.scss` so a widget directly rendered without Form(Control) gets the fix as well?
- Use `padding-inline-end` instead of `padding-right`?
- Add `box-sizing: border-box;` to `.mykn-stackcts` to make sure the padding has as little effect on the width as possible?